src/abstracts/Base.js

```
@@ -76,7 +76,12 @@ function getRefs(instance, element) {
 function getChildren(instance, element, components) {
   const children = Object.entries(components).reduce((acc, [name, ComponentClass]) => {
     const selector = `[data-component="${name}"]`;
-    const elements = Array.from(element.querySelectorAll(selector));
+    let elements = Array.from(element.querySelectorAll(selector));
+
+    // If no child component found with the default selector, the name must be a DOM selector
+    if (elements.length === 0) {
+      elements = Array.from(element.querySelectorAll(name));
```

Should maybe throw error if no results for qsa(name) ? To improve the dev experience

---

Actually, this is a feature 🙂 
It fails silently on purpose to load async components only if elements are found in the DOM.